db/compaction_iterator.h

```diff
@@ -21,6 +21,38 @@
 
 namespace rocksdb {
 
+class SnapshotListFetchCallback {
+ public:
+  SnapshotListFetchCallback(uint64_t _snap_refresh_nanos,
+                            size_t _every_nth_key = 1024)
```

Comment:
Is `_every_nth_key` fixed in production, and the argument is only there for testing? Is there a way to remove this argument?

Reply:
Correct. I needed that only for testing.